elm.json

```diff
@@ -1,7 +1,7 @@
 {
     "type": "package",
     "name": "PaackEng/paack-ui",
-    "summary": "Paack design system",
+    "summary": "Paack's Design System applied in components for Elm UI",
```

Comment:
I think by definition, Design system already implies that is in components 🤔 , so maybe "Paack's Design System for Elm UI"